superset/security/manager.py

```diff
@@ -1160,6 +1160,18 @@ def get_rls_ids(self, table: "BaseDatasource") -> List[int]:
         ids.sort()  # Combinations rather than permutations
         return ids
 
+    @staticmethod
+    def raise_for_user_activity_access(user_id: int) -> None:
+        id_ = int(g.user.get_id())
```

Comment:
this may fail, `g.user` is not always safe

Reply:
also, why not just `g.user.id` ?

Reply:
Good point about `g.user`, will address. And good question about `g.user.get_id()` - I just noticed `g.user.get_id()` is used fairly prevalently, so I assumed it was the preferred way. `user.id` sounds better here IMO